### PR TITLE
feat: show BBB recordings

### PR DESCRIPTION
**Pull request by MeghalBisht**
reqd: fossasia/open-event-server#7838

Screenshot from 2021-03-19 05-01-59

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/events/view/videoroom-form.hbs [hunk 1]
```diff
@@ -134,6 +134,18 @@
       {{t 'Add'}}
     </button>
   </div>
+  {{#if (and this.recordings (eq this.data.stream.videoChannel.provider 'bbb'))}}
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.hbs:137]: Instead, simply check this.videoRecordings

File: .env.example [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #MAPBOX_ACCESS_TOKEN = "Access Token"
-API_HOST=https://open-event.dokku.fossasia.org
+API_HOST=http://localhost:5000
```
Comment (iamareebjamal) [inline .env.example]: Revert

Comment (MeghalBisht) [reply .env.example]: yes, done already

**Comment by mariobehling** [comment]
Thank you, it is good that the feature is there. Right now we do not have a place where we have an overview of recordings. We need to think this through, but for now it seems ok to have it here.

**Pull request merged**